sklearn/base.py

```diff
@@ -923,7 +923,7 @@ def get_feature_names_out(self, input_features=None):
         feature_names_out : ndarray of str objects
             Transformed feature names.
         """
-        check_is_fitted(self, "_n_features_out")
```

Comment:
I'm a bit curious about the role of `_n_features_out` here. I wonder if it's among the right attributes for `check_is_fitted`.